packages/manager/apps/pci-workflow/src/pages/executions/Executions.page.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+  Datagrid,
+  DataGridTextCell,
+  Headers,
+  useDatagridSearchParams,
+  useProject,
+  useProjectUrl,
+} from '@ovhcloud/manager-components';
+import { ODS_SPINNER_SIZE } from '@ovhcloud/ods-components';
+import { OsdsBreadcrumb, OsdsSpinner } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { useHref, useParams } from 'react-router-dom';
+import ExecutionStatusComponent from '@/components/execution-status.component';
+import { useWorkflowExecutions } from '@/api/hooks/useExecutions';
+import { TWorkflowExecution } from '@/api/data/region-workflow';
+
+export default function Executions() {
+  const { t: tExecution } = useTranslation('executions');
+
+  const { projectId, workflowId } = useParams();
+  const { data: project } = useProject();
+  const hrefProject = useProjectUrl('public-cloud');
+  const backHref = useHref('..');
+
+  const {
+    pagination,
+    setPagination,
+    sorting,
+    setSorting,
+  } = useDatagridSearchParams();
+
+  const {
+    data: { executions, workflowName },
+    isPending,
+  } = useWorkflowExecutions(projectId, workflowId, {
+    pagination,
+    sorting,
+  });
+
+  const columns = [
```

Comment:
to avoid sonar issues could you please move column definition outside of the component ?

Reply:
Done